packages/inter-protocol/src/stakeFactory/stakeFactoryKit.js

```diff
@@ -358,7 +358,7 @@ const helperBehavior = {
     helper.assertVaultHoldsNoMinted();
     seat.exit();
 
-    return 'Your RUNstake is closed; thank you for your business.';
+    return 'Your stakeFactory is closed; thank you for your business.';
```

Comment:
Messages like this don't play a critical security role like invitation descriptions, but strictly speaking, this is also a breaking contract API change.

Reply:
The API just says that it returns a string. Anything depending on parsing that string is making unwarranted assumptions about the API.

Reply:
We have quite a few tests that depend on the contents of these strings. That seemed like an unwarranted assumption to me too, but that seems to be the current design. I guess it doesn't matter that much...